src/utils/lodash.tsx

```diff
@@ -0,0 +1,116 @@
+/**
+ * This file implements lodash alternatives used by components
+ *
+ * Importing the lodash library causes problems when building Mística as an ES module.
+ * Importing lodash-es causes problems when running ssr tests (not sure why)
+ *
+ * Once Mística gets migrated to a proper ES module, we can consider to remove this file and use lodash-es
+ */
+
+type Debounced<T> = T & {cancel: () => void};
+
+export const debounce = <T extends (...args: Array<any>) => any>(
+    func: T,
+    wait: number,
+    options: {
+        leading?: boolean;
+        maxWait?: number;
+    } = {}
+): Debounced<T> => {
+    let debounceTimeoutId: ReturnType<typeof setTimeout> | undefined;
+    let maxWaitTimeoutId: ReturnType<typeof setTimeout> | undefined;
+    let currentArgs: Parameters<T>;
+    let isLeading = true;
+
+    const debounced = (...args: Parameters<T>) => {
+        if (debounceTimeoutId) {
+            clearTimeout(debounceTimeoutId);
+        }
+
+        currentArgs = args;
+
+        if (!maxWaitTimeoutId && options.maxWait) {
+            maxWaitTimeoutId = setTimeout(() => {
+                func(...currentArgs);
+                maxWaitTimeoutId = undefined;
+                clearTimeout(debounceTimeoutId);
+            }, options.maxWait);
+        }
+
+        if (isLeading && options.leading) {
+            isLeading = false;
+            func(...args);
+            return;
+        }
+
+        debounceTimeoutId = setTimeout(() => {
+            func(...args);
+            if (maxWaitTimeoutId) {
+                clearTimeout(maxWaitTimeoutId);
+            }
+            debounceTimeoutId = undefined;
+            maxWaitTimeoutId = undefined;
+            // eslint-disable-next-line testing-library/await-async-utils
```

Comment:
false positive. I've created an issue in js-toolbox to track these problems

https://github.com/Telefonica/js-toolbox/issues/new